nuxeo-ai-core/src/test/java/org/nuxeo/ai/configuration/PersistedConfigurationServiceTest.java

```diff
@@ -115,13 +120,31 @@ public void shouldGetAllConfigsByClass() throws IOException {
 
     @Test
     public void iCanPropagateConfiguration() throws InterruptedException {
-        messageReceivedLatch = new CountDownLatch(1);
-        aiConfigurationService.set(threshold);
-        if (!messageReceivedLatch.await(5, TimeUnit.SECONDS)) {
-            fail("message not received in 5s");
+        int thresholdSize = ((ThresholdComponent) Framework.getRuntime()
+                                                           .getComponent(
+                                                                   "org.nuxeo.ai.configuration.ThresholdComponent")).typeThresholds.size();
+        // TODO: AICORE-366
+        // messageReceivedLatch = new CountDownLatch(1);
+        aiConfigurationService.set(thresholdFolder);
```

Comment:
It's not enough for our tests?

Reply:
I would prefer in general to have a lock rather than timeout